app/models/application_record.rb

```diff
@@ -3,6 +3,11 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
 
+  scope :with_array_order, lambda { |ids, column = :id, column_type = :bigint|
```

Comment:
Do we need this is `ApplicationRecord`?

Reply:
For this BZ no. Now we need it **only** for `CustomButtonSet` model. But it looks to me as useful functionality so I put it here because we have user defined-order on more places.

What do you think? Where I should put it?  Should I put it only to `CustomButtonSet` model and then eventually in the future put it to the mixin?

Reply:
@bdunne Are you ok with leaving this in `ApplicationRecord`? Seems like it wouldn't hurt to have it there.

Reply:
I guess so

Reply:
@lpichler I'm interpreting @bdunne response to 👎 for keeping this in `ApplicationRecord`. Let's move it to `CustomButtonSet`. We can always elevate it later is another model needs it.

Reply:
thanks, just small correction for what I wrote - it was about `CustomButton` model